host-configuration/docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Introduction
+
+AGL Development Kit allows developers and integrators to rebuild complete bootable images
+from source code, as well as developing new applications for AGL.
+
+It leverages Docker capabilities to propose a unified environment suitable for most development tasks:
```

Comment:
This intro is a copy/paste of sdk-devkit README file.
I would update it so that it will better reflect host configuration stuff.
You can reuse the 1st part of this doc : https://en.opensuse.org/LinuxAutomotive